compilers/src/make.js

```diff
@@ -1,9 +1,9 @@
 const fs = require('fs')
 
-const domains = fs.readFileSync('../../domains', 'utf8').split('\n')
+const domains = fs.readFileSync('domains', 'utf8').split('\n')
 
-fs.appendFile('hostfile', '##COINHIVE DOMAINS\n')
+fs.appendFile('hostfile', '##COINHIVE DOMAINS\n', ()=>{})
```

Comment:
Are you sure this won't accidentally write the header on the second line? Have you done checks?

Reply:
Checked, it's a new line.